habitat-lab/habitat/config/benchmark/rearrange/play_stretch.yaml

```diff
@@ -15,10 +15,10 @@ habitat:
     actions:
       arm_action:
         type: "ArmAction"
-        arm_controller: "ArmRelPosKinematicReducedActionStretch"
+        arm_controller: "ArmRelPosReducedActionStretch"
         grip_controller: "MagicGraspAction"
-        arm_joint_mask: [1,0,0,0,1,1,1,1,1,1]
-        arm_joint_dimensionality: 10
+        arm_joint_mask: [1,0,0,0,1,1,1,1,0,0]
```

Comment:
The last two elements here correspond to `joint_head_pan` and `joint_head_tilt`, right? Does this mean that the policies we train will no longer move the head camera?

Reply:
This is just for the play script so we can ignore it and merge the branch?

Reply:
Sure